src/components/v2/V2Create/forms/RulesForm/index.tsx

```diff
@@ -22,27 +24,54 @@ import {
 import FormItemLabel from '../../FormItemLabel'
 import { getBallotStrategyByAddress } from 'constants/v2/ballotStrategies/getBallotStrategiesByAddress'
 
-export default function RulesForm({ onFinish }: { onFinish: VoidFunction }) {
+interface InitialValues {
```

Comment:
**Question:** would all these `InitialValues` interfaces be better named just `RulesFormFields` and so on?

Reply:
It's a good idea - so the reason I didn't do this initially is because there is already a `FundingFormFields` in `FundingForm/index.tsx`, so to keep everything consistent I called InitialValues

Reply:
Can we just create new ones for `RulesFormFields` and `TokenFormFields` then reuse the `FundingFormFields` one? Having `InitialValues` as an interface or type just feels quite misleading

Reply:
The crossover between `FundingFormFields` and `InitialValues` wasn't there so i've just dropped them since we can implicitly still retrieve the data from the memo's anyway. It wasn't really apply much value as is